remote/remote_state_test.go

```diff
@@ -18,10 +21,20 @@ func TestToTerraformInitArgs(t *testing.T) {
 			"bucket":  "my-bucket",
 			"key":     "terraform.tfstate",
 			"region":  "us-east-1",
-		},
+
+			"s3_bucket_tags": map[string]interface{}{
+				"team":    "team name",
+				"name":    "Terraform state storage",
+				"service": "Terraform"},
+
+			"dynamotable_tags": map[string]interface{}{
+				"team":    "team name",
+				"name":    "Terraform state storage",
+				"service": "Terraform"}},
 	}
 	args := remoteState.ToTerraformInitArgs()
 
+	// must not contain s3_bucket_tags or dynamotable_tags
```

Comment:
Does this test check the S3 tags were actually created?

Reply:
No, it just calls ToTerraformInitArgs.

It seems there is no test at all that actually creates the bucket, if you guys add one later I'd be happy to add the check for the tags.

Reply:
A bunch of the tests in [integration_test.go](https://github.com/gruntwork-io/terragrunt/blob/master/test/integration_test.go) create an S3 bucket. See the fixtures folder.

Reply:
Ok found and extended it to check for tags of both s3 and dynamo db.

The test case I changed runs on my machine, if I let the whole test run, I get this, but I assume it's not related to my changes:

```
[terragrunt] [/var/folders/3h/j7zf9_595vb97k60z03zt2nh0000gp/T/terragrunt-test258914403/fixture-output-all/env1/app1] 2018/06/15 11:42:21 Running command: terraform apply -input=false -auto-approve -var terraform_remote_state_s3_bucket="terragrunt-test-bucket-mgqpz9"
Copying configuration from "file:///Users/bernhard/go/src/github.com/gruntwork-io/terragrunt/test/fixture-download/hello-world"...
Initializing modules...
- module.hello
- module.remote
  Getting source "github.com/gruntwork-io/terragrunt.git//test/fixture-download/hello-world?ref=v0.9.9"
Error downloading modules: Error loading modules: subdir "test/fixture-download/hello-world" not found
--- FAIL: TestLocalDownloadWithHiddenFolder (0.14s)
	integration_test.go:817: Failed to run Terragrunt command 'terragrunt apply --terragrunt-non-interactive --terragrunt-working-dir fixture-download/local-with-hidden-folder' due to error: Hit multiple errors:
		exit status 1
```